### ISSUE TITLE
About the setting of hyperparameters

### ISSUE THREAD

**Issue by Study-gsl** [fannie1208/CaNet#2]
Thank you for your great work. May I ask if the hyper-parameters in the run.sh file are optimal?

**Comment by fannie1208** [comment]
Hi, the hyper-parameters in the run.sh file are optimal and we run them on an RTX 3090.

**Comment by Study-gsl** [comment]
I am very grateful for your answer! I have some confusion and hope you can help me. 
For the implementation of the regularization term in equation 11, is logK derived by considering the prior distribution as 1/K? Meanwhile, what part of its code implementation is it (model.py line182)? Can you help me provide a corresponding intuitive explanation? 
Looking forward to your answer! Thank you again!

**Comment by fannie1208** [comment]
Hi, thank you for your question!
Yes, logK is derived by considering the prior distribution as 1/K. 
Since logK is a constant and will not contribute to the gradient, in the code we omit its implementation.
